src/api/app/controllers/source_controller.rb

```diff
@@ -48,16 +48,8 @@ def index
   end
 
   def projectlist
-    # list all projects (visible to user)
-    output = Rails.cache.fetch(['projectlist', Project.maximum(:updated_at), Relationship.forbidden_project_ids]) do
-      dir = Project.pluck(:name).sort
-      output = ''
-      output << "<?xml version='1.0' encoding='UTF-8'?>\n"
-      output << "<directory>\n"
-      output << dir.map { |item| "  <entry name=\"#{::Builder::XChar.encode(item)}\"/>\n" }.join
-      output << "</directory>\n"
-    end
-    render xml: output
+    @projects = Project.pluck(:name).sort
```

Comment:
Please rename @projects to @project_names. That way it's clear that it's a list of names and not objects